# Taller 3.2

**Pruebas PostMan:**

* Create measurement:
![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/POST.png)

Se crea correctamente le medicion.

* Get measurements:
![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/GET%20ALL.png)

Se adquieren todas las mediciones correctament. En el momento del get, solo hay uno.

* Get measurement:
![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/GET%20ID.png)

Se adquiere la medicion el id=1 con el metodo QueryParams.

* Update measurement:
![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/UPDATE%201.png)

Se intenta a hacer un put en la medicion con id=1 pero aparece un error que no puede corregir.
Sin embargo, se puede observar que hubo un update de la medicion como se muestra a continuacion

![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/UPDATE%202.png)
Intentamos arreglar el error pero no pudimos

* Delete measurement:
![alt text](https://github.com/YeiHong/TallerDjango-Vistas/blob/master/imagenes/DELETE.png)

Se elimina correctamente la medicion.
